biomedsheets/__main__.py

```diff
@@ -59,9 +59,9 @@ def run(self):
     def load_sheet_schema(self):
         """Load the bundled SheetSchema"""
         print("Loading bundled BioMed Sheet JSON Schema...", file=sys.stderr)
-        return SheetSchema.load_from_string(
-            pkg_resources.resource_stream("biomedsheets", "data/sheet.schema.json").read().decode()
-        )
+        ref = importlib_resources.files("biomedsheets").joinpath("data/sheet.schema.json")
+        inputf = ref.open("rb")
+        return SheetSchema.load_from_string(inputf.read().decode())
```

Comment:
there are also convenience shorthands for reading textfiles, see https://docs.python.org/3/library/importlib.resources.html#functional-api

Reply:
or use a contextmanager (`with ref.open("rb") as inputf:`)